test/e2e/aws.go

```diff
@@ -48,6 +58,19 @@ func (a *awsTestHelper) makeCredentialsSecret(namespace string) *corev1.Secret {
 	}
 }
 
+func (a *awsTestHelper) externalDNS(testExtDNSName, hostedZoneID, hostedZoneDomain string, credsSecret *corev1.Secret) operatorv1alpha1.ExternalDNS {
```

Comment:
```suggestion
func (a *awsTestHelper) externalDNS(name, zoneID, zoneDomain string, credsSecret *corev1.Secret) operatorv1alpha1.ExternalDNS {
```
Purely cosmetic:
- just to not be redundant: we are already in test and the method is already called `externalDNS`
- `zoneID` and `zoneDomain` and just shorter and for some providers they are not called "hosted"

Reply:
Done